resources/views/product/create.blade.php

```diff
@@ -0,0 +1,125 @@
+@extends('layouts/app')
+
+@section('page-title')
+  @lang('products.menu_products')
+@stop
+
+@section('section-title')
+  @lang('products.item_add_product')
+@stop
+@section('errors-message')
+    @include('common.errors')
+@stop
+
+@section('susscess-message')
+    @include('common.success')
+@stop
+@section('page-content')
+  <div class="x_content">
+    <form class="form-horizontal form-label-left" role="form" method="POST" action="{{ route('product.store') }}">
+        {!! csrf_field() !!} 
+         <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="name">@lang('products.label_name_product') <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input id="name" class="form-control col-md-7 col-xs-12" data-validate-length-range="2" data-validate-words="1" name="name" placeholder="@lang('products.label_name_product')" required="required" type="text" >
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="categories">@lang('products.label_categories_product')
+              <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <select autofocus name="category_id">
+                  @foreach($categories as $category)
+                    <option value="{{ $category->id }}">{{ $category->name }}</option>
+                  @endforeach
+              </select>
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="number"> @lang('products.label_price_product') <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input type="number" id="number" name="price" required="required" data-validate-min="1,200000" class="form-control col-md-7 col-xs-12" placeholder="000.000 vnd">
```

Comment:
[rank B] you have to define min, max values as common constant.
All placeholder need define in language file too.